src/openforms/conf/ci.py

```diff
@@ -55,8 +55,11 @@
 # Django privates
 SENDFILE_BACKEND = "django_sendfile.backends.development"
 
-# Two factor auth
-TWO_FACTOR_FORCE_OTP_ADMIN = False
+# Two factor auth - disable admin enforcement in CI
+MAYKIN_2FA_ALLOW_MFA_BYPASS_BACKENDS = AUTHENTICATION_BACKENDS + [
+    "django_webtest.backends.WebtestUserBackend",
+    "django_webtest.backends.WebtestUserWithoutPermissionsBackend",
+]
```

Comment:
this can be removed again and instead the decorator itself should be enough on the test itself